app/pandas.ipynb

```diff
@@ -145,18 +218,31 @@
   },
   {
    "cell_type": "code",
-   "execution_count": 11,
-   "outputs": [],
-   "source": [
-    "# Write your code here"
-   ],
+   "source": "df[df[\"Age\"] > 18]",
```

Comment:
The KeyError indicates that the 'Age' column is not found. This might be due to a previous operation that altered the DataFrame. Ensure that the 'Age' column exists in the DataFrame before attempting to filter it.